test/integration/BUILD

```diff
@@ -489,6 +489,21 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
```

Comment:
Doesn't _necessarily_ need to be split up from the circuit breaker tests, but they don't really share much 🤷‍♂️